bundles/org.eclipse.equinox.p2.ui/src/org/eclipse/equinox/internal/p2/ui/dialogs/TrustCertificateDialog.java

```diff
@@ -853,9 +854,6 @@ private static class QuestionDialog extends MessageDialog {
 
 		public QuestionDialog(Shell shell, String title, String message, String labelAccept, String labelReject) {
```

Comment:
We can also revert the addition of this class from my POV, I added it due to the handling for `SWT.ON_TOP`.